docs/dashboard/apis.md

```diff
@@ -467,9 +467,386 @@ Responses
+| policyData | object | Described below                 | Required |
+
+Policy Data
+
+| Param        | Type  | Description                                                                     | Required |
+|--------------|-------|---------------------------------------------------------------------------------| -------- |
+| url          | string | Webhook URL which will be sent a POST request with the webhook data, and userOp | Required |
+
+Responses
+
+> **_200 OK_**
+
+```json
+{
+    "statusCode": 200,
+    "message": "Paymaster Policy created successfully",
+    "data": {
+            "_id": "xxxxx-xxxx-xxxx-xxxx-xxxxx",
+            "name": "webhook rule",
+            "policyType": "WEBHOOK",
+            "organisationId": "xxxxx-xxxx-xxxx-xxxx-xxxxx",
+            "paymasterId": "xxxxx-xxxx-xxxx-xxxx-xxxxx",
+            "createdBy": "xxxxx-xxxx-xxxx-xxxx-xxxxx",
+            "chainId": 84532,
+            "active": true,
+            "policyData": {
+                "url": "https://www.google.com"
+             },
+       }     
+}
+```
+
+> **_401 Unauthorized_**
+
+```json
+{
+    "statusCode": 401,
+    "message": "Auth token and API key is required in the headers"
+}
+```
+
+
+#### 10. Add a "Wallet Deployment" rule to the paymaster
+
+> **_POST Request_**
+
+URL: https://paymaster-dashboard-backend.prod.biconomy.io/api/v2/public/sdk/paymaster-policy
+
+Parameters
+
+Header
+
+| Param     | Type   | Description                        | Required |
+| --------- | ------ | ---------------------------------- | -------- |
+| authToken | string | Token unique to every user account | Required |
+| apiKey    | string | API Key Associated with dApp       | Required |
+
+Body
+
+| Param      | Type   | Description             | Required |
+|------------|--------|-------------------------| -------- |
+| name       | string | Unique Policy name      | Required |
+| policyType | string | Pass "WALLET_DEPLOYMENT" | Required |
+| policyData | object | Pass empty object (`{}`)  | Required |
+
+Responses
+
+> **_200 OK_**
+
+```json
+{
+    "statusCode": 200,
+    "message": "Paymaster Policy created successfully",
+    "data": {
+            "_id": "xxxxx-xxxx-xxxx-xxxx-xxxxx",
+            "name": "wallet deployment rule",
+            "policyType": "WALLET_DEPLOYMENT",
+            "organisationId": "xxxxx-xxxx-xxxx-xxxx-xxxxx",
+            "paymasterId": "xxxxx-xxxx-xxxx-xxxx-xxxxx",
+            "createdBy": "xxxxx-xxxx-xxxx-xxxx-xxxxx",
+            "chainId": 84532,
+            "active": true,
+            "policyData": {
+                "operator": "walletDeployment"
+             },
+       }     
+}
+```
+
+> **_401 Unauthorized_**
+
+```json
+{
+    "statusCode": 401,
+    "message": "Auth token and API key is required in the headers"
+}
+```
+
+
+
+#### 11. Get all rules for a paymaster (spending limit, webhook, wallet deployment, whitelisted contracts)
+
+> **_GET Request_**
```

Comment:
are there any filters on this API for policy type or contract

Reply:
Nope